test_runner/fixtures/pageserver/allowed_errors.py

```diff
@@ -106,6 +106,11 @@ def scan_pageserver_log_for_errors(
     ".*startup_reconcile: Could not scan node.*",
     # Tests run in dev mode
     ".*Starting in dev mode.*",
+    # Tests that stop endpoints & use the storage controller's neon_local notification
+    # mechanism might fail (neon_local's stopping and endpoint isn't atomic wrt the storage
+    # controller's attempts to notify the endpoint).
+    ".*reconciler.*Local notification hook failed.*",
```

Comment:
Can we change that log message to
```suggestion
    ".*reconciler.*neon_local notification hook failed.*",
```

Reply:
d55c59201